sdk/identity/azure-identity/azure/identity/aio/_credentials/environment.py

```diff
@@ -52,6 +57,16 @@ def __init__(self, **kwargs: "Any") -> None:
                 **kwargs
             )
 
+        if self._credential:
+            _LOGGER.info("Environment is configured for %s", self._credential.__class__.__name__)
+        else:
+            expected_variables = set(EnvironmentVariables.CERT_VARS + EnvironmentVariables.CLIENT_SECRET_VARS)
+            set_variables = [v for v in expected_variables if v in os.environ]
+            if set_variables:
+                _LOGGER.warning("Incomplete environment configuration. Set variables: %s", ", ".join(set_variables))
```

Comment:
This sentence feels semantically ambiguous to me, it can mean either:

- [imperative mood] Please set variables: 
- [past participle] Variables that are already set:

Reply:
@dbradish-microsoft

Reply:
#15974